### ISSUE TITLE
[Accessibility] [Global] [Accordion] Basic Accordion> Accordion buttons are not tagged in a 'list' programmatically.

### ISSUE THREAD

**Issue by silvalaura** [cengage/react-magma#1998]
[A11Y-502](https://jira.cengage.com/browse/A11Y-502)

**Environment:**
Windows 11/Chrome 137/ JAWS 2025
Windows 11/Edge 136/ NVDA 2025
Mac 15/Safari/ VoiceOver
 
**Steps to Reproduce:**
1. Open the URL- https://react-magma.cengage.com/version/4.9.1/api/tooltip/
2. Turn On screen reader (JAWS/NVDA).
2. Launch 'Accordion' component page from left navigation.
4. On the 'Accordion' page, under 'Basic Usage', navigate through list of accordion buttons using arrow keys, and verify.
 
 
**Actual Result:**
The 'Section 1', 'Section 2', 'Section 3' buttons are not programmatically tagged in an unordered list for screen reader users.
 
Refer to screencast:
https://app.screencast.com/LQTcYsT6bgwNC
 
**Note**: Same issue is reproducible for all other accordion buttons.
 
**Expected Result:**
The collapsible/expandable accordion buttons should be programmatically tagged in an unordered list.
 
**User Impact:**
Screen reader users wouldn't realize how many collapsible/expandable items are there in the accordion section.
 
WCAG Guideline: 1.3.1 Info & Relationships (A)
 
**Bug Fix Recommendation:**
Wrap the collapsible/expandable accordion buttons inside an unordered list using ```<ul> <li> </li> <li> </li> <li> </li> </ul>``` markup.

**Comment by olehnoskov-cengage** [comment]
**Tested links:**

[V5](https://storybook-preview-dev--upbeat-sinoussi-f675aa.netlify.app/?path=/story/accordion--default)
[V4](https://storybook-preview-v4-dev--upbeat-sinoussi-f675aa.netlify.app/?path=/story/accordion--default)

Tested on Windows 11/10 - Google Chrome, Firefox, Edge + NVDA
Tested on MacOs - Google Chrome, Firefox, Safari + VoiceOver

<img width="668" height="747" alt="Image" src="https://github.com/user-attachments/assets/28031313-8ea6-437b-ba41-e9528b975a8e" />
<img width="968" height="503" alt="Image" src="https://github.com/user-attachments/assets/6f5d8183-c1d6-47a6-b8f0-dd79e6952634" />

Everything works well! ✅
